scaffolder-templates/basic-workflow/template.yaml

```diff
@@ -89,20 +94,48 @@ spec:
           type: string
           pattern: '^([a-zA-Z][a-zA-Z0-9]*)([-._][a-zA-Z0-9]+)*$'
           description: Maven Artifact ID
+          default: onboarding
         version:
           title: Project version
           type: string
           default: 1.0.0-SNAPSHOT
           description: Maven version, also used to tag the generated artifacts
-
+    - title: Provide information about the Deployment environment
+      required:
+        - ci
+        - clusterDomain
+        - namespace
+      properties:
+        ci:
+          title: Select a CI method
+          type: string
+          description: This action will create a CI pipeline for your application based on chosen method
+          default: tekton
+          enum:
+            - none
+            - tekton
+          enumNames:
+            - None
+            - Tekton
+        clusterDomain:
+          title: Cluster domain
+          type: string
+          default: cluster-jkrpp.dynamic.redhatworkshops.io
+          description: Cluster domain
```

Comment:
should we specify in this context this cluster domain is for the tekton pipeline so it won't be mixed with being also the cluster to run the workflows?

Reply:
Ok, even if having workflows and tekton on different clusters is something we have never experiment so far (probably the ArgoCD will need its own cluster configuration, in this multi-cluster deployment)